typings/blpapijs/blpapijs.d.ts

```diff
@@ -0,0 +1,52 @@
+import events = require('events');
```

Comment:
I don't think we should put this in the `typings/` directory since it is used by other tools to place typings here.  Maybe we should explore another directory such as `etc/` or something.